src/pages/patientView/timeline/legacy.js

```diff
@@ -166,6 +166,7 @@ export function buildTimeline(params, caseIds, patientInfo, clinicalDataMap, cas
     window.pvTimeline =
         window.pvTimeline
             .splitByClinicalAttributes("Treatment", ["SUBTYPE", "AGENT"])
+            .splitByClinicalAttributes("Treatment", ["TREATMENT_TYPE", "AGENT"])
```

Comment:
So, now we want to split both by `subtype` and `treatment_type`, right?

Just making sure that this is the desired behavior :)